coti.py

```diff
@@ -21,26 +23,6 @@ def format_decimal(number):
     return str(number).replace(".", "").replace(",", ".")
 
 
-def vision():
-    soup = None
-    try:
-        soup = BeautifulSoup(
-            requests.get('https://www.visionbanco.com', timeout=10,
-                         headers={'user-agent': 'Mozilla/5.0'}, verify=False).text, "html.parser")
-
-        efectivo = soup.select('#efectivo')[0]
-        compra = efectivo.select(
-            'table > tr > td:nth-of-type(2) > p:nth-of-type(1)')[0].get_text().replace('.', '')
-        venta = efectivo.select(
-            'table > tr > td:nth-of-type(3) > p:nth-of-type(1)')[0].get_text().replace('.', '')
-    except requests.ConnectionError:
-        compra, venta = 0, 0
-    except:
-        compra, venta = 0, 0
-
-    return Decimal(compra), Decimal(venta)
-
-
```

Comment:
Vision is removed here because this PR sits on top of the other PR that removes it.